renovate.json

```diff
@@ -1,5 +1,20 @@
 {
+  "timezone": "Europe/Berlin",
```

Comment:
Any particular reason for Berlin - maybe UTC would be more universal?

Reply:
Indeed, fixed in 7f775ffb